modules/coreos-layering-configuring-on.adoc

```diff
@@ -57,32 +96,31 @@ spec:
-         dnf clean all && \
-         ostree container commit
-    imageBuilder: # <5>
+        FROM configs AS final
+        RUN rpm-ostree install tree && \
+            ostree container commit
+    imageBuilder: # <3>
       imageBuilderType: PodImageBuilder
-    baseImagePullSecret: # <6>
+    baseImagePullSecret: # <4>
       name: global-pull-secret-copy
-    renderedImagePushspec: image-registry.openshift-image-registry.svc:5000/openshift/os-image:latest  # <7>
-    renderedImagePushSecret: # <8>
+    renderedImagePushspec: image-registry.openshift-image-registry.svc:5000/openshift/os-image:latest  # <5>
+    renderedImagePushSecret: # <6>
       name: builder-dockercfg-7lzwl
-  buildOutputs: # <9>
+  buildOutputs: # <7>
     currentImagePullSecret:
       name: builder-dockercfg-7lzwl
 ----
-<1> Specifies the machine config pool to deploy the custom layered image.
-<2> Specifies the Containerfile to configure the custom layered image. You can specify multiple build stages in the Containerfile.
-<3> Specifies the architecture of the image to be built. You must set this parameter to `noarch`.
-<4> Specifies the build stage as final. This field is required and applies to the last image in the build.
-<5> Specifies the name of the image builder to use. You must set this parameter to `PodImageBuilder`.
-<6> Specifies the name of the pull secret that the MCO needs in order to pull the base operating system image from the registry.
-<7> Specifies the image registry to push the newly-built custom layered image to. This can be any registry that your cluster has access to. This example uses the internal {product-title} registry.
-<8> Specifies the name of the push secret that the MCO needs in order to push the newly-built custom layered image to the registry.
-<9> Specifies the secret required by the image registry that the nodes need in order to pull the newly-built custom layered image. This should be a different secret than the one used to push the image to your repository.
+<1> Specifies the name of the machine config pool associated with the nodes where you want to deploy the custom layered image.
+<2> Specifies the Containerfile to configure the custom layered image.
+<3> Specifies the name of the image builder to use. This must be `PodImageBuilder`.
+<4> Specifies the name of the pull secret that the MCO needs in order to pull the base operating system image from the registry.
+<5> Specifies the image registry to push the newly-built custom layered image to. This can be any registry that your cluster has access to. This example uses the internal {product-title} registry.
+<6> Specifies the name of the push secret that the MCO needs in order to push the newly-built custom layered image to that registry.
+<7> Specifies the secret required by the image registry that the nodes need in order to pull the newly-built custom layered image. This should be a different secret than the one used to push the image to your repository.
```

Comment:
We don't have this field anymore in the `v1` API. Please see above comment

Reply:
According to [David's comment above](https://github.com/openshift/openshift-docs/pull/87730#issuecomment-2663957330)  and my testing, 4.18 uses vialpha1